src/scripts/main.js

```diff
@@ -5,4 +5,23 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const current = document.querySelector('tr');
+
+people.forEach(({ name: fullName, died, born, sex }) => {
+  const century = Math.ceil(died / 100);
+
+  const table = document.createElement('tr');
+
+  current.parentElement.append(table);
+
+  table.insertAdjacentHTML(
```

Comment:
Use 'td' elements for table data instead of 'th'. 'th' elements are for table headers.